ext/ldap/ldap.stub.php

```diff
@@ -4,281 +4,281 @@
 /**
- * @param resource $link_identifier
+ * @param resource $ldap
  * @return resource|false
  */
-function ldap_mod_replace_ext($link_identifier, string $dn, array $entry, array $servercontrols = []) {}
+function ldap_mod_replace_ext($ldap, string $dn, array $entry, array $controls = []) {}
 
-/** @param resource $link_identifier */
-function ldap_mod_del($link_identifier, string $dn, array $entry, array $servercontrols = []): bool {}
+/** @param resource $ldap */
+function ldap_mod_del($ldap, string $dn, array $entry, array $controls = []): bool {}
 
 /**
- * @param resource $link_identifier
+ * @param resource $ldap
  * @return resource|false
  */
-function ldap_mod_del_ext($link_identifier, string $dn, array $entry, array $servercontrols = []) {}
+function ldap_mod_del_ext($ldap, string $dn, array $entry, array $controls = []) {}
 
-/** @param resource $link */
-function ldap_errno($link): int {}
+/** @param resource $ldap */
+function ldap_errno($ldap): int {}
 
-/** @param resource $link */
-function ldap_error($link): string {}
+/** @param resource $ldap */
+function ldap_error($ldap): string {}
 
 function ldap_err2str(int $errno): string {}
 
-/** @param resource $link_identifier */
-function ldap_compare($link_identifier, string $dn, string $attribute, string $value, array $servercontrols = []): bool|int {}
+/** @param resource $ldap */
+function ldap_compare($ldap, string $dn, string $attribute, string $value, array $controls = []): bool|int {}
 
 
 #ifdef LDAP_CONTROL_PAGEDRESULTS
 /**
- * @param resource $link
+ * @param resource $ldap
  * @deprecated since 7.4
  */
-function ldap_control_paged_result($link, int $pagesize, bool $iscritical = false, string $cookie = ""): bool {}
+function ldap_control_paged_result($ldap, int $pagesize, bool $iscritical = false, string $cookie = ""): bool {}
 
 /**
- * @param resource $link
+ * @param resource $ldap
  * @param resource $result
  * @param string $cookie
  * @param int $estimated
  * @deprecated since 7.4
  */
-function ldap_control_paged_result_response($link, $result, &$cookie = null, &$estimated = null): bool {}
+function ldap_control_paged_result_response($ldap, $result, &$cookie = null, &$estimated = null): bool {}
 #endif
 
 #if (LDAP_API_VERSION > 2000) || defined(HAVE_ORALDAP)
-/** @param resource $link_identifier */
-function ldap_rename($link_identifier, string $dn, string $newrdn, string $newparent, bool $deleteoldrdn, array $servercontrols = []): bool {}
+/** @param resource $ldap */
+function ldap_rename($ldap, string $dn, string $new_rdn, string $new_parent, bool $delete_old_rdn, array $controls = []): bool {}
 
 /**
- * @param resource $link_identifier
+ * @param resource $ldap
  * @return resource|false
  */
-function ldap_rename_ext($link_identifier, string $dn, string $newrdn, string $newparent, bool $deleteoldrdn, array $servercontrols = []) {}
+function ldap_rename_ext($ldap, string $dn, string $newrdn, string $newparent, bool $deleteoldrdn, array $controls = []) {}
```

Comment:
```suggestion
function ldap_rename_ext($ldap, string $dn, string $new_rdn, string $new_parent, bool $delete_old_rdn, array $controls = []) {}
```
For consistency with the ldap_rename() change.